redfish/rules/dell_redfish_1.17.0.yml

```diff
@@ -0,0 +1,431 @@
+    Type: health
+  - Name: chassis_systemembedded1_pciedevices_status_state
+    Pointer: /Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/PCIeSlots
+  Properties:
+  - Name: chassis_systemembedded1_pcieslots_slots_status_state
+    Pointer: /Slots/{slot}/Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/Power
+  Properties:
+  - Name: chassis_systemembedded1_power_powersupplies_redundancy_status_health
+    Pointer: /PowerSupplies/{powersupply}/Redundancy/{redundancy}/Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_power_powersupplies_redundancy_status_state
+    Pointer: /PowerSupplies/{powersupply}/Redundancy/{redundancy}/Status/State
+    Type: state
+  - Name: chassis_systemembedded1_power_powersupplies_status_health
+    Pointer: /PowerSupplies/{powersupply}/Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_power_powersupplies_status_state
+    Pointer: /PowerSupplies/{powersupply}/Status/State
+    Type: state
+  - Name: chassis_systemembedded1_power_redundancy_status_health
+    Pointer: /Redundancy/{redundancy}/Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_power_redundancy_status_state
+    Pointer: /Redundancy/{redundancy}/Status/State
+    Type: state
+  - Name: chassis_systemembedded1_power_voltages_status_health
+    Pointer: /Voltages/{voltage}/Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_power_voltages_status_state
+    Pointer: /Voltages/{voltage}/Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/PowerSubsystem
+  Properties:
+  - Name: chassis_systemembedded1_powersubsystem_powersupplyredundancy_status_health
+    Pointer: /PowerSupplyRedundancy/{powersupplyredundancy}/Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_powersubsystem_powersupplyredundancy_status_state
+    Pointer: /PowerSupplyRedundancy/{powersupplyredundancy}/Status/State
+    Type: state
+  - Name: chassis_systemembedded1_powersubsystem_status_health
+    Pointer: /Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_powersubsystem_status_state
+    Pointer: /Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/PowerSubsystem/PowerSupplies/{psu-slot}
+  Properties:
+  - Name: chassis_systemembedded1_powersubsystem_powersupplies_status_health
+    Pointer: /Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_powersubsystem_powersupplies_status_state
+    Pointer: /Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/PowerSubsystem/PowerSupplies/{psu-slot}/Metrics
+  Properties:
+  - Name: chassis_systemembedded1_powersubsystem_powersupplies_metrics_status_health
+    Pointer: /Status/Health
+    Type: health
+  - Name: chassis_systemembedded1_powersubsystem_powersupplies_metrics_status_state
+    Pointer: /Status/State
+    Type: state
+- Path: /redfish/v1/Chassis/System.Embedded.1/Sensors/{fan.embedded}

```

Comment:
Do not include `.` in placeholders.

Reply:
I have reworked the rules for redfish v1.17 based on your comments.